src/tests/utils/ApiTestSteps.ts

```diff
@@ -139,6 +139,15 @@ export async function getSessionAndVerifyKey(sessionId: any, tableName: string,
 	}
 }
 
+export async function getSessionAndVerifyKeyExists(sessionId: any, tableName: string, key: string) {
```

Comment:
@jackmardell please could we have a return type for this function?
![image](https://github.com/govuk-one-login/ipv-cri-bav-api/assets/40401118/2ca6ab0d-a6f1-43b2-8591-d7eebcb25db0)

Also `sessionId` could probably be `string`?

Reply:
Yes please - `Promise<void>` is perfect. Just lets the consumer of the function know not to expect anything to be returned